BrainPortal/db/migrate/20170913185428_add_container_index_to_tool_configs.rb

```diff
@@ -0,0 +1,5 @@
+class AddContainerIndexToToolConfigs < ActiveRecord::Migration
```

Comment:
Add data migration to populate existing TCs with the new attribute filled by parsing the `containerhub_image_name` content; you will have to rewrite the migration to contain an explicit `up` and `down` method

Reply:
Addressed in https://github.com/aces/cbrain/pull/649/commits/f1313b2fad911425b7dee7f0686510892d46deae